webapp/src/actions/index.js

```diff
@@ -13,16 +13,6 @@ export function startMeeting(channelId, rootId = '', force = false, topic = '')
                 window.open(meetingURL);
             }
         } catch (error) {
-            let m = error.message;
-            if (error.message && error.message[0] === '{') {
-                const e = JSON.parse(error.message);
-
-                // Error is from Zoom API
-                if (e && e.message) {
-                    m = '\nZoom error: ' + e.message;
-                }
-            }
```

Comment:
Hmm this error may have some useful context for why the meeting couldn't start, like something going on with the user's zoom account. I'm thinking this can stay

Reply:
@mickmister Reverted the changes to display generic error message

Reply:
It's mainly when we enter the "Error is from Zoom API" block that I'm thinking we should show the provided error. If the others from our server are more cryptic/low-level, then let's only show the error from `error.message` if the error came from Zoom, meaning we are in the "Error is from Zoom API" block here

Reply:
@mickmister Just to clarify, if we are in the  "Error is from Zoom API" block, then we show the error from the API and show a generic error otherwise. Right?

Reply:
@raghavaggarwal2308 Yes that sounds right to me

Reply:
@mickmister Updated